src/test/java/seedu/address/logic/commands/ExitCommandTest.java

```diff
@@ -1,10 +1,11 @@
 package seedu.address.logic.commands;
 
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.logic.commands.ExitCommand.MESSAGE_EXIT_ACKNOWLEDGEMENT;
+import static seedu.address.logic.commands.sharkie.ExitCommand.MESSAGE_EXIT_ACKNOWLEDGEMENT;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.logic.commands.sharkie.ExitCommand;
```

Comment:
To avoid having to import each command for each test can just move all the commandtests into the proper packages also (seedu.address.logic.commands.sharkie, seedu.address.logic.commands.people).